extensions/amp-base-carousel/1.0/base-carousel.js

```diff
@@ -127,9 +129,14 @@ function BaseCarouselWithRef(
           placeholder. When a slide's slot is unrendered, the slide
           automatically gets unslotted and gets CanRender=false w/o any extra
           state management code.
+
+          Note: We naively display all slides for mixedLength as multiple
```

Comment:
We can do that only for a non-loop case, right? Or do you see a way to do with `loop` too?